src/drive/web/modules/views/OnlyOffice/Toolbar/index.jsx

```diff
@@ -16,12 +16,16 @@ import FileName from 'drive/web/modules/views/OnlyOffice/Toolbar/FileName'
 import ReadOnly from 'drive/web/modules/views/OnlyOffice/Toolbar/ReadOnly'
 import Sharing from 'drive/web/modules/views/OnlyOffice/Toolbar/Sharing'
 import useUpdateDocumentTitle from 'drive/web/modules/views/useUpdateDocumentTitle'
+import useUpdateDocumentFavicon from '../../useUpdateDocumentFavicon'
```

Comment:
Why relative `../..` path ?

Reply:
I need to fix my auto import in VS Code. I'll correct that :)

Reply:
If you've fixed that later, you should squash the fix commit with this one.